nvflare/tool/code_pre_installer/README.md

```diff
@@ -0,0 +1,119 @@
+# NVFLARE Code Pre-Installer
+
+A tool to pre-install NVFLARE application code and libraries.
+
+## Overview
+
+The code pre-installer handles:
+- Installation of application code
+- Installation of shared libraries
+- Site-specific customizations
+- Python package dependencies
+
+## Directory Structure
+
+Expected application code zip structure:
+```
+app_code.zip
+├── app_code/
+│   ├── meta.json           # Application metadata
```

Comment:
Our app inside a Job does not have a meta.json that specify the name of it.

Maybe we can just do this:

app_code.zip
|---------copy/
|
|---------shared/


what ever inside "copy" is copied over as is,
so when user is preparing this app_code.zip,
they can put their job specific codes there:
copy/job1/xxx
what ever in "shared" is adding to the PYTHONPATH.